build/tools/Automation/Automation.csproj

```diff
@@ -22,4 +22,11 @@
     <Folder Include="Commons\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="SmartAnalyzers.CSharpExtensions.Annotations" Version="4.2.8">
```

Comment:
It looks like we can just update the `Directory.Build.props` file that contains a reference to this package and will automatically be imported in all projects.

Reply:
you are right. thanks for the suggestion